en/messages.json

```diff
@@ -330,6 +348,37 @@
     "popupSignBackInBody_mask_v2": {
         "message": "To create a new email mask when you’re asked for your email, open the context menu (right-click or control-click) in an email field. From there you can create and use a new mask right away."
     },
+    "popupCreateCustomMaskTitle": {
+        "message": "Create a new custom mask"
+    },
+    "popupCreateCustomMaskBody": {
+        "message": "All you need to do is make up and share a unique mask that uses your custom subdomain — the mask will be generated automatically. Try “shop@customsubdomain.mozmail.com” next time you shop online, for example."
+    },
+    "popupCreateCustomFormMaskInputLabel": {
+        "message": "Enter email mask prefix"
+    },
+    "popupCreateCustomFormMaskInputPlaceholder": {
+        "message": "e.g. “coffee”"
+    },
+    "popupCreateCustomFormMaskInputDescription": {
+        "message": "@$SUBDOMAIN$.mozmail.com",
+        "description": "",
+        "placeholders": {
+            "subdomain": {
+                "content": "$1",
+                "example": "example )"
+            }
+        }        
+    },
+    "popupCreateCustomFormMaskBlockPromosChecboxLabel": {
+        "message": "Block promotional emails"      
+    },
+    "popupCreateCustomFormMaskBlockSubmitButton": {
+        "message": "Generate Mask"      
```

Comment:
should it be lower case `mask`?

Reply:
Great catch! Revised in 47e6fc94f8ab2fdc620e148c6c1611ef1d1665e7